src/core/border_router/routing_manager.cpp

```diff
@@ -72,7 +72,7 @@ RoutingManager::RoutingManager(Instance &aInstance)
     , mDiscoveredPrefixInvalidTimer(aInstance, HandleDiscoveredPrefixInvalidTimer)
     , mDiscoveredPrefixStaleTimer(aInstance, HandleDiscoveredPrefixStaleTimer)
     , mRouterAdvertisementCount(0)
-    , mLastRouterAdvertisementSendTime(TimerMilli::GetNow())
+    , mLastRouterAdvertisementSendTime(TimerMilli::GetNow() - kMinDelayBetweenRtrAdvs)
```

Comment:
@simonlingoogle Have you tried removing the two lines added to multi network test in https://github.com/openthread/openthread/pull/7473:
https://github.com/openthread/openthread/blob/main/tests/scripts/thread-cert/border_router/test_multi_thread_networks.py#L111-L112

Reply:
I haven't tried that. 
I think it's still fine to wait for stabilize.

Reply:
verified that removing `delaying 15 seconds` will cause the test to fail again. 
So, we will still keep the delaying.